### ISSUE TITLE
consider list namespaces command

### ISSUE THREAD

**Issue by Brofessional** [mar-file-system/mad#16]
It might be helpful to list the namespaces in a repo. There is a lot of noise in config files we don't want to see always.

**Comment by jti-lanl** [comment]
I agree it would be useful to have a tool to do this.  Meanwhile, I believe doing an ‘ls’ of the top-level fuse-mount will give you a list of namespaces.

On Oct 13, 2020, at 7:16 PM, Shane Goff <notifications@github.com<mailto:notifications@github.com>> wrote:



It might be helpful to list the namespaces in a repo. There is a lot of noise in config files we don't want to see always.

**Comment by Brofessional** [comment]
One small problem I run into often during testing is I'm not on a node with the fuse mount running.

**Comment by Brofessional** [comment]
It might also be useful for displaying the namespaces in a situation where you have more than one config file laying around

**Comment by jti-lanl** [comment]
Yep, useful.

There is a config-dumping tool that is useful for scripts, etc, where you just want to parse the config-file to extract some info.

**Comment by Brofessional** [comment]
Creating a list command was really easy, so I threw it in.
